components/card/card.css

```diff
@@ -0,0 +1,164 @@
+.card{
+    /* border : 3px solid red; */
+    border-radius: 1rem;
```

Comment:
cards are visually appealing. everything is great. Go ahead and merge the branch.